testsuite/tests/win-unicode/Makefile

```diff
@@ -0,0 +1,25 @@
+BASEDIR=../..
+LIBRARIES=unix
+ADD_COMPFLAGS= \
+	-I $(OTOPDIR)/otherlibs/$(UNIXLIBVAR)unix \
+	-strict-sequence -safe-string -w A -warn-error A
+LD_PATH=$(TOPDIR)/otherlibs/$(UNIXLIBVAR)unix
+ADD_BYTERUN_FLAGS=-I $(OTOPDIR)/otherlibs/win32unix
+MAIN_MODULE=mltest
+
+ifeq ($(OS),Windows_NT)
+.PHONY: test
+test:
+	@. ./conf.sh && \
```

Comment:
Yes, this is a leftover from the original tests provided by Clément Franchini.  I will clean this up.